clients/fides-js/src/lib/consent.tsx

```diff
@@ -55,6 +52,9 @@ const constructLocation = (
   if (geoLocation.country && geoLocation.region) {
     return `${geoLocation.country}-${geoLocation.region}`;
   }
+  if (geoLocation.country) {
```

Comment:
I don't think the experience API currently supports filtering when only country is provided. Is this something we need to add on the backend? If so, this would lead to a case where multiple experiences could be returned from the API

Reply:
Unless we plan on adding just country to this enum? https://github.com/ethyca/fides/blob/main/src/fides/api/ops/models/privacy_notice.py#L20

Reply:
Ah- we'll need to handle that case then. In reality we can't geolocate *that* accurately every time... can you open a ticket and let's check in with @pattisdr and team?

Reply:
It's just currently filtering on the regions in that list - but stepping back for a second, what do we do if the united states is what is filtered on. How do we decide which experience to show then?

Reply:
We'll have to have some fallbacks & defaults defined for when we don't get exact matches... I did have a good chat about some edge cases like this with @mfbrown yesterday. We need a way to guarantee we get a defined experience no matter what location is determined (including "unknown"), which'll require some updates 👍

Reply:
Ticket here- https://github.com/ethyca/fides/issues/3300